engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+    "binding-decrease-view-distance": "کاهش فاصله ی دید",
+    "binding-forwards": "رو به جلو",
+    "binding-frob": "هدف استفاده",
+    "binding-hide-hud": "اچ یو دی مخفی",
+    "binding-increase-view-distance": "افزایش فاصله ی دید",
+    "binding-jump": "پرش",
+    "binding-left": "گردش از چپ",
+    "binding-nui-editor": "NUI Editor",
+    "binding-nui-skin-editor": "ان. یو. آی ادیتور اسکین",
+    "binding-pause": "منوی انتخاب در حین بازی",
+    "binding-right": "گردش از راست",
+    "binding-show-online-players": "نمایش بازیکنان انلاین",
+    "binding-toggle-speed-permanently": "دکمه ی سرعت دایمی",
+    "binding-toggle-speed-temporarily": "دکمه ی سرعت موقتی",
+    "binding-use-item": "استفاده از آیتم نگه داشته شده",
+    "bloom-effect": "بلووم                        ⁠⁠",
+    "bobbing": "حرکت به بالا و پایین",
+    "camera-blur": "تیرگی دوربین",
+    "camera-blur-max": "تیرگی دوربین در بالاترین حد",
+    "camera-blur-normal": "تیرگی دوربین در حد طبیعی",
+    "camera-blur-off": "خاموش کردن تیرگی دوربین",
+    "camera-blur-some": "Some",
+    "camera-details-settings": "تنظیمات جزییات گرافیکی دوربین",
+    "camera-film-grain": "بافت فیلم",
+    "camera-setting": "تنظیمات دوربین",
+    "camera-setting-cinematic": "سینمایی",
+    "camera-setting-normal": "تنظیمات معمولی دوربین",
+    "camera-setting-smooth": "نرم",
+    "category-behavior": "دسته بندی عمومی رفتار",
+    "category-general": "دسته بندی عمومی رفتار",
+    "category-interaction": "اثر متقابل",
+    "category-inventory": "دسته بندی -بازبینی",
+    "category-mouse": "موش واره",
+    "category-movement": "حرکت",
+    "category-nui": "NUI",
+    "change-keybind-popup-message": "is already bound to an action. Rebind anyway?",
+    "change-keybind-popup-title": "کلید هم اکنون مهیا است",
+    "clamp-lighting": "مهار نور",
+    "cloud-shadows": "سایه های ابر",
+    "connecting-to": "Connecting to",
+    "connection-failed": "قطع اتصال",
+    "could-not-connect-to-server": "نمی تواند به کارساز ( سرور) وصل شود",
+    "credits": "اعتبارها",
+    "create-game": "خلق",
+    "create-game-title": "خلق بازی",
+    "create-preview": "پیش دید کردن",
+    "custom-servers": "بر حسب عادت",
+    "deactivate-module": "غیر فعال",
+    "death-screen-title": "شما مرده اید",
+    "default-binding": "از بین رفتن اتصال",
+    "delete-game": "حذف کردن",
+    "dev-tools-bt-editor": "نمایشگر درختی ویرایش رفتاری",
+    "dev-tools-nui-editor": "ویرایشگر صفحه ان. یو. ال",
+    "dev-tools-skin-editor": "ویرایشگر اسکین ان.یو. ال",
+    "dev-tools-title": "ابزار توسعه دهنده",
+    "dialog-cancel": "لغو کردن",
+    "dialog-ok": "درست است",
+    "disable-all-modules": "از کار انداختن همه",
+    "download-module": "دانلود",
+    "downloading-server-list": "دانلود لیست کارساز..",
+    "drop-item": "قطع مطلب",
+    "edit-server": "ویرایش",
+    "edit-server-title": "اضافه/ویرایش  کارساز (سرور)",
+    "enter-username-message": "Hello and welcome to Terasology's multiplayer!\nWould you like to use a non-default username?",
```

Comment:
This is untranslated - can you wrap this up just so the language is 100% finished? [4]